app/(main)/profile/page.tsx

```diff
@@ -37,13 +37,27 @@ import { useEnsName } from 'wagmi';
 import { mainnetConfigForENS } from '@/providers/chain-config';
 import { mainnet } from 'viem/chains';
 
-import { AiAgent } from '@/types';
-
+const ITEMS_PER_PAGE = 100;
```

Comment:
Why 100 items? Can we show 10 per page? I see that all bets depends on this number, maybe we can fix the "all bets number problem" in another PR

Reply:
Yeah, all the filters depends on the "All Bets", and the "All Bets" is the result from getting the query result from The Graph and then applying a filter in the FE, so our simulated pagination sometimes behave in an odd way.

Anyway, the other tabs are filters, so they'll never be `> All bets`